js/toolbar/annotation-tool.js

```diff
@@ -24,11 +24,16 @@ function AnnotationTool(parent){
         var btnType = this.getAttribute("data-type") || "";
 
         if(_self.curType == btnType){
-            return;
+            console.log('same');
+            if (btnType == 'CURSOR') {
+                // we dont want to do anything if the cursor is clicked again
+                return;
+            }
+            btnType = 'CURSOR';
         }
 
 
-        _self.dispatchEvent("drawingStop");
+        _self.dispatchEvent("drawingStop", {'buttonType': btnType});
```

Comment:
Change this back to whatever it was before

Reply:
Changed